superset/viz.py

```diff
@@ -258,14 +266,14 @@ def process_query_filters(self):
         merge_extra_filters(self.form_data)
         utils.split_adhoc_filters_into_base_filters(self.form_data)
 
-    def query_obj(self):
+    def query_obj(self) -> Dict[str, Any]:
         """Building a query object"""
         form_data = self.form_data
         self.process_query_filters()
         gb = form_data.get("groupby") or []
         metrics = self.all_metrics or []
         columns = form_data.get("columns") or []
-        groupby = []
+        groupby: List[str] = []
         for o in gb + columns:
             if o not in groupby:
                 groupby.append(o)
```

Comment:
nit: this bit bothers me. Can we this for something like 
```python
groupby = list(set(gb + columns))`
```

Reply:
True, will throw that in.